### PR TITLE
IODEMO: Kill with -9, Stop via SIGINT

### PR DESCRIPTION
**Pull request by avildema**
https://github.com/Mellanox/hpc-mtt-conf/issues/638

kill via -9
stop via SIGINT

### CODE REVIEW

File: test/apps/iodemo/run_io_demo.sh [hunk 1]
```diff
@@ -694,6 +694,13 @@ make_scripts()
 			    for pid in \$(list_pids_with_role \${tag})
 			    do
 			        echo "Stopping process \${pid}"
+			        kill -INT \${pid}
+			    done
+			    ;;
+			kill)
+			    for pid in \$(list_pids_with_role \${tag})
+			    do
+			        echo "Killing process \${pid}"
```
Comment (yosefe) [inline test/apps/iodemo/run_io_demo.sh:703]: minor: to align, in line 704, do: `kill -KILL \${pid}`

**Comment by yosefe** [comment]
bot:pipe:retest

**Review by yosefe — APPROVED** [review]
pls squash

**Comment by alinask** [comment]
Need to also add -kill to parse_args() https://github.com/openucx/ucx/blob/master/test/apps/iodemo/run_io_demo.sh#L523 (and to the usage() function too please)
(@avildema @yosefe )

**Pull request merged**